querydsl-mongodb/src/main/java/org/mongodb/morphia/geo/QPoint.java

```diff
@@ -0,0 +1,38 @@
+package org.mongodb.morphia.geo;
+
+import javax.annotation.Nullable;
+
+import com.querydsl.core.types.PathMetadata;
+import com.querydsl.core.types.dsl.BooleanExpression;
+import com.querydsl.core.types.dsl.EntityPathBase;
+import com.querydsl.core.types.dsl.PathInits;
+import com.querydsl.mongodb.morphia.MorphiaExpressions;
+
+/**
+ * {@code QPoint} is an adapter type for Point to use geo spatial querying features of Mongodb
+ *
+ * @author jmoghisi
+ *
+ */
+public class QPoint extends EntityPathBase<Point> {
```

Comment:
I didn't fully understand this point, when is EntityPathBase intended to be used?